src/components/activities/searchPatientActivity/SearchPatientActivity.tsx

```diff
@@ -60,6 +67,43 @@ const SearchPatientActivity: FunctionComponent<TProps> = ({
     return has(formik.touched, fieldName) ? get(formik.errors, fieldName) : "";
   };
 
+  const handleScrollToElement = () => {
+    const elem = resultsRef.current;
+    window.scrollTo({
+      top: (elem) ? elem.offsetTop - 200 : 0,
+      behavior: 'smooth',
+    }); 
+  }
```

Comment:
This function could be parameterized in order to make it of general use, so we can reuse it throughout the project when we need it. What do you think? I'd put it in `src/libraries/uiUtils/handleScrollToElement.ts`. Also, it resolves the [OP-310](https://openhospital.atlassian.net/browse/OP-310). Yay!